### DISCUSSION TITLE
Image Transforms Quality as a number.

### DISCUSSION THREAD

**Discussion by pimago** [craftcms/cms#9622] [Ideas] [answered]
In the documentation the CP image quality setting is described as a number from 0-100. But it's a limited set of predefined settings. A config setting to change the pre defined quality settings to an actually number would be much more flexible if needed. The difference and quality drop from Max to Very High is just too strong.
I know I could override the quality setting inside the template. But that's not convenient for clients.

**Comment by brandonkelly** [comment]
The plan is to make this possible in Craft 4.

**Reply by pimago** [reply]
Cool. Thanks for the quick reply.

**Reply by pimago** [reply]
Are you still planning on adding this feature?

**Reply by brandonkelly** [reply]
Sorry, slipped through the cracks, but I’ve just added it for 4.3.

**Comment by brandonkelly** [comment, marked as answer]
This has been added for Craft 4.3 (#11884) 🎉

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week with this change!
